CMakeLists.txt

```diff
@@ -127,25 +127,22 @@ add_subdirectory(src)
 if (MSVC)
     if (${CMAKE_VERSION} VERSION_GREATER 3.26)
```

Comment:
Should we make this code run on all systems. I find it quite useful to have the build folder runable and not require an install first.